setup/security/authorization/index.md

```diff
@@ -68,17 +68,41 @@ accounts.
 ### Accounts
 In the dark ages (before Fiat), only accounts could be restricted. Because
 Clouddriver is the source of truth for accounts, Fiat reaches out to Clouddriver
-to gather the list of available accounts. To add access restrictions to an
-account, update your `halconfig` to include the `requiredGroupMembership` field.
+to gather the list of available accounts. 
+
+There are two types of access restrictions to an account, `READ` and `WRITE`. Users must have
+matching roles to all the `READ` permissions of an account to view the account's cloud resources.
```

Comment:
@stewchen 
Would it be correct to say "User must have `READ` access to an account to view..." rather than "..must have matching roles to all the `READ` permissions..."
(What is the importance of the "matching roles" part?)

Reply:
I was trying to imply that a user would need to have all `READ` roles if there were multiple, to be able to read. I'll just separate these into two sentences hah.